app/scripts/controllers/transactions/index.js

```diff
@@ -1219,13 +1219,24 @@ export default class TransactionController extends EventEmitter {
    *  params instead of allowing this method to generate them
    * @param options
    * @param options.estimatedBaseFee
+   * @param actionId
    * @returns {txMeta}
    */
   async createSpeedUpTransaction(
     originalTxId,
     customGasSettings,
     { estimatedBaseFee } = {},
+    actionId,
```

Comment:
Is this something we could add to the options param?

Reply:
Can you plz detail, this point is not clear to me.

Reply:
there is the existing `options` param to pass arbitrary and optional values

currently we have
```
   * @param options
   * @param options.estimatedBaseFee
   * @param actionId
   * @returns {txMeta}
   */
  async createSpeedUpTransaction(
    originalTxId,
    customGasSettings,
    { estimatedBaseFee } = {},
    actionId,
```
we could update this to

```
   * @param options
   * @param options.actionId
   * @param options.estimatedBaseFee
   * @returns {txMeta}
   */
  async createSpeedUpTransaction(
    originalTxId,
    customGasSettings,
    { actionId, estimatedBaseFee } = {},
```

Using an `options` object param can help us reduce the number of arguments. Some benefits to keeping the argument size small is better readability and less friction to call the function. 

If we had optional params in the middle, we might end up filling params with nulls to call the method. Example should we expand this method to include an additional param 
```
TransactionsController.createSpeedUpTransaction(txId, customGas, null, null, true)
```

Reply:
Hey @digiwand : third parameter is `newTxMetaProps` I would prefer to keep actionId separate.